src/test/scala/sparkz/core/api/http/PeersApiRouteSpec.scala

```diff
@@ -58,7 +58,9 @@ class PeersApiRouteSpec extends AnyFlatSpec
   }
 
   it should "connect to peer" in {
-    val body = HttpEntity("localhost:8080".asJson.toString).withContentType(ContentTypes.`application/json`)
+    val body = HttpEntity(
+      """{"address": "localhost:8080"}"""
```

Comment:
Just a reminder: when we will apply this change to SDK, need to update STF utils and documentation (sidechainApi.yaml and doc.horizen) with the new body scheme